LICENSE.md

```diff
@@ -16,19 +16,19 @@ Each contributor licenses you to do everything with this software that would oth
 
 ## License Reversion
 
-Each contribution to this software is licensed under this license for three years from the date it is contributed to the software. After that three year period the contribution is licensed under the MIT license.
+Each contribution to this software is licensed under this license for three years from the date it is contributed to the software. After that three year period the contribution is licensed under the [Blue Oak Model License v 1.0.0](https://blueoakcouncil.org/license/1.0.0).
 
 ## Code of Conduct
 
-You must use this software in compliance with the ml5.js [Code of Conduct](README.md) as it exists at the time of your use. This includes cooperating with any investigations of your compliance with the ml5.js Code of Conduct by the Code of Conduct Comittee or any contributor or their designees.
+You must use this software in compliance with the ml5.js [Code of Conduct](README.md) as it exists at the time of your use. This includes cooperating with any investigations of your compliance with the ml5.js Code of Conduct.
 
 ## Notices
 
-You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to [canonical URL for the license].
+You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to https://github.com/ml5js/Code-of-Conduct/blob/main/LICENSE.md.
```

Comment:
@bomanimc this is where I'd like to switch to `ml5js.org/license`.

Reply:
Nonblocking comment / suggestion:
* We should remember to update the license file in the ml5-library repo and the package.json

Reply:
Working on both now!

Reply:
Looked into this, and realized I forgot that we had already mapped this URL in https://github.com/ml5js/ml5-website/pull/172. Currently, /license maps to the root directory of our CoC repository. Long-term, I wonder if this actually provides more flexibility for restructuring, since changes to the path of the license would require an update of the redirect URL. What do you think @shiffman?

Reply:
Yes, this 100% makes sense! Let's link directly to this repo. We also want to emphasize the Code of Conduct's relationship to the license so it makes more sense conceptually to do so! So I think we are all set. I'll update the link in this document now.